scripts/bfasst/compare_waveforms/Interface/waveform_interface.py

```diff
@@ -0,0 +1,65 @@
+import yaml
+
+"""A function to check if there are multiple verilog files in a design or not. Used later in parsing stages due to different logic being needed."""
+
+
+def check_multiple_files(design):
+    with open(design.yaml_path) as fp:
+        design_props = yaml.safe_load(fp)
+
+    multiple_files = False
+
+    for k, v in design_props.items():
```

Comment:
Since you never use `v`, it would be better to use `for k in design_props.keys():`